flycheck.el

```diff
@@ -11238,6 +11239,22 @@ See URL `http://www.ruby-doc.org/stdlib-2.0.0/libdoc/yaml/rdoc/YAML.html'."
   :modes yaml-mode
   :next-checkers ((warning . cwl)))
 
+(flycheck-def-config-file-var flycheck-yamllintrc yaml-yamllint ".yamllint"
+  :safe #'stringp)
+
+(flycheck-define-checker yaml-yamllint
+  "A YAML syntax checker using YAMLLint.
+See URL `https://github.com/adrienverge/yamllint'."
+  :command (
+            "yamllint" "-f" "parsable" source
+            (config-file "-c" flycheck-yamllintrc))
```

Comment:
From https://yamllint.readthedocs.io/en/stable/quickstart.html#running-yamllint it seems that yamllint supports checking standard input; could we use this?  This way we don't have to create temporary files.

Reply:
At the moment of writing of original code it didn't support stdin and author didn't have plan to implement it. I will update my PR to use stdin.